### PR TITLE
SVCPLAN-3219 Updates to README

### PR DESCRIPTION
**Pull request by bsper2**

### CODE REVIEW

**Comment by bsper2** [comment]
This addresses concerns in issue #26

File: README.md [hunk 1]
```diff
@@ -50,17 +50,24 @@ Logging of user processes and open network connections is disabled by default. T
...
+
+See this table to summarize when you need `qualys::escalated_scans` and/or `qualys_eus_reporting::enabled`:
+
+|Server on <br />EUS Release| Using Redhat<br />repo URLs |Recommended Setting|
+| --- | --- | --- |
+| True | True | `profile_audit::qualys::escalated_scans: true`<br />`profile_audit::qualys_eus_reporting::enabled: false`|
+| False | False | Default (both false) |
+| True | False | `profile_audit::qualys::escalated_scans: true`<br />`profile_audit::qualys_eus_reporting::enabled: true` |
+| False | True | Default (both false) |
```
Comment (billglick) [inline README.md:70]: Perhaps change the ordering here. The order seemed very random to me.

```suggestion
| False | False | Default (both false) |
| False | True | Default (both false) |
| True | False | `profile_audit::qualys::escalated_scans: true`<br />`profile_audit::qualys_eus_reporting::enabled: true` |
| True | True | `profile_audit::qualys::escalated_scans: true`<br />`profile_audit::qualys_eus_reporting::enabled: false`|
```

**Review by billglick — APPROVED** [review]
This looks good.

Only comment was I found the table hard to parse. Perhaps changing the ordering would help a bit.

But I'm also fine as is.

File: README.md (hunk 1 above)
Comment (bsper2) [reply README.md]: Haha, it was in an order that's easy to confirm I had the each permutation of True/False. 1) Both True 2) Both False 3) and 4) T/F flipped with each other. But I can see how your suggestion is easier for someone actually trying to USE the table, so I updated to use your ordering.
